sequencer.c

```diff
@@ -3481,6 +3513,9 @@ static int do_merge(struct repository *r,
 					 "GIT_COMMITTER_DATE=%s", date);
 			free(date);
 		}
+		if (opts->ignore_date)
+			argv_array_pushf(&cmd.env_array,
+					 "GIT_AUTHOR_DATE=%ld", time(NULL));
```

Comment:
Convenient :)

Reply:
AFAIK, --date sets AUTHOR TZ to UTC. But we are using local TZ in the logic. Should be changed ASAP.